tinywl/tinywl.c

```diff
@@ -786,6 +784,12 @@ static void xdg_toplevel_request_move(
 	 * client, to prevent the client from requesting this whenever they want. */
 	struct tinywl_toplevel *toplevel = wl_container_of(listener, toplevel, request_move);
 	begin_interactive(toplevel, TINYWL_CURSOR_MOVE, 0);
+
+	// TODO
+	struct wlr_box surface_box;
+	wlr_xdg_surface_get_geometry(toplevel->xdg_toplevel->base, &surface_box);
+	wlr_scene_node_set_position(&toplevel->shadow->node, surface_box.x - toplevel->shadow->blur_sigma,
```

Comment:
With the suggested changes in the other comment, this shouldn't be needed due to the positions being parent relative.

I.e. parent tree pos = (30, 30), the shadow tree pos of (0, 0) would be (30, 30)